src/main/java/uk/gov/hmcts/ccd/domain/service/common/CaseDataAccessControl.java

```diff
@@ -0,0 +1,11 @@
+package uk.gov.hmcts.ccd.domain.service.common;
+
+import uk.gov.hmcts.ccd.domain.model.definition.CaseDetails;
+
+import java.util.Optional;
+
+public interface CaseDataAccessControl {
```

Comment:
Can we move this to below package:
uk.gov.hmcts.ccd.domain.service.casedataaccesscontrol

Reply:
So it would be the interface and two implementation classes.